### PR TITLE
[Docs]refactor: Merge Build and Releases page inside Contributing section

### LINKED ISSUE #6057: [Docs] Merge Build and Releases page inside Contributing section
#### Current State
We have a different section for build and releases, Since this is something particularly needed by developers, putting it inside the Contributing would be useful

Screen Shot 2022-08-18 at 12 31 54

#### Desired State
Screen Shot 2022-08-18 at 12 34 52

NOTE: Please take care of the URL, it should be similar to other sections present inside contributing.
---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by sambit-sankalp**
Signed-off-by: Sambit Sankalp <sambit.sankalp.official@gmail.com>

**Description**
- Merged the Build and Releases page inside the Contributing section
- The screenshot of the achieved state is below
image

This PR fixes #6057 

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0e66a464bfdc72cb7e57c84c363572b00dd8adee at: https://63026df1650679353ff69d69--meshery-docs-preview.netlify.app

**Review by asubedy — APPROVED** [review]
Great. LGTM 👍

**Review by abhiyant-10 — APPROVED** [review]

**Comment by sambit-sankalp** [comment]
Hey @abhiyant-10 @asubedy, just pushed the latest commits into this branch, so that later it will not create merge conflicts.

**Comment by l5io** [comment]
🚀 Preview for commit c03e1b2c883d43fa63dcb0ab5e974df8791ad6eb at: https://630404e8a24c2c16219b66fa--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/ci-build-and-release.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ---
-layout: default
```
Comment (leecalcote) [inline docs/pages/project/contributing/ci-build-and-release.md:2]: What's driving the need for this layout change? Are the other contributing pages using `page` as their layout, too?

File: docs/_data/toc.yml [hunk 2]
```diff
@@ -25,10 +25,10 @@
           url: project/contributing/contributing-ui
         - title: End-to-End Testing
           url: project/contributing/contributing-cypress
+        - title: Build & Release (CI)
```
Comment (leecalcote) [inline docs/_data/toc.yml:28]: Please sort these menu items alphabetically.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/project/contributing/ci-build-and-release.md (hunk 1 above)
Comment (sambit-sankalp) [reply docs/pages/project/contributing/ci-build-and-release.md:2]: Yes, the other contributing pages using `page` as their layout, so I changed it.

File: docs/_data/toc.yml (hunk 2 above)
Comment (sambit-sankalp) [reply docs/_data/toc.yml:28]: Sure

**Comment by l5io** [comment]
🚀 Preview for commit 21310fb25d7dec0f34d893ba6c6ce93e58f3c14e at: https://6305eff7c3e95f6507fd371f--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b4224b087f7f9ac3944f1939520c2cecf1ca6f2d at: https://6305f15d15558a62bb32fb0e--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f4d6db0f0923cfcd2e2c975d9e91defc75ad4210 at: https://63093630a1e19b0c5a5bf965--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/ci-build-and-release.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/project/contributing/ci-build-and-release.md:2]: Gotcha. Thank you, @sambit-sankalp 👍

**Review by leecalcote — APPROVED** [review]
Thanks. Good work, @sambit-sankalp 👍

**Pull request merged**